demo/admin/src/products/ManufacturerFilter.tsx

```diff
@@ -0,0 +1,79 @@
+import { gql, useQuery } from "@apollo/client";
+import Autocomplete from "@mui/material/Autocomplete";
+import TextField from "@mui/material/TextField";
+import { GridFilterInputValueProps, GridFilterOperator } from "@mui/x-data-grid-pro";
+import * as React from "react";
+import { useIntl } from "react-intl";
+
+import { GQLManufacturersFilterQuery, GQLManufacturersFilterQueryVariables } from "./ManufacturerFilter.generated";
+
+const manufacturersQuery = gql`
+    query ManufacturersFilter($offset: Int!, $limit: Int!, $search: String) {
+        manufacturers(offset: $offset, limit: $limit, search: $search) {
+            nodes {
+                id
+                name
+            }
+            totalCount
+        }
+    }
+`;
+
+// Source: https://mui.com/x/react-data-grid/filtering/customization/#multiple-values-operator
+function ManufacturerFilter({ item, applyValue }: GridFilterInputValueProps) {
+    const intl = useIntl();
+    const [search, setSearch] = React.useState<string | undefined>(undefined);
+
+    const { data } = useQuery<GQLManufacturersFilterQuery, GQLManufacturersFilterQueryVariables>(manufacturersQuery, {
+        variables: {
+            offset: 0,
+            limit: 10,
+            search: search,
+        },
+    });
+
+    // source https://mui.com/material-ui/react-autocomplete/
+    return (
+        <Autocomplete
+            size="small"
+            options={data?.manufacturers.nodes ?? []}
+            autoHighlight
+            value={item.value ? item.value : null}
+            filterOptions={(x) => x} // disable local filtering
+            isOptionEqualToValue={(option, value) => {
+                // does only highlight the selected value in options-list but does not trigger getOptionLabel-Call
+                return option.id == value;
+            }}
+            getOptionLabel={(option) => {
+                // option.name is not set if loaded from filters-model, small delay replacing id with name because of loading
+                return option.name ?? data?.manufacturers.nodes.find((item) => item.id === option)?.name ?? option;
+            }}
+            onChange={(event, value, reason) => {
+                // value can't be "{ id: value.id, name: value.name }" because value is sent to api
+                applyValue({ id: item.id, operatorValue: "equals", value: value ? value.id : undefined, columnField: "manufacturer" });
+            }}
+            renderInput={(params) => (
+                <TextField
```

Comment:
We do not support MUIs `TextField` please use `InputBase` instead, see:
- [Search Autocomplete Story](https://storybook.comet-dxp.com/?path=/story/stories-components-toolbar-search-autocomplete--search-autocomplete)
- [SearchAutoComplete.stories.tsx](https://github.com/vivid-planet/comet/blob/main/storybook/src/docs/components/Toolbar/stories/SearchAutoComplete.stories.tsx)

Reply:
@johnnyomair Should we add an import restriction for this?

Reply:
As far as I know we have no need for MUIs TextField

Reply:
I created a task for this: https://vivid-planet.atlassian.net/browse/COM-1158
FYI @thomasdax98